### PR TITLE
[BACKLOG-37004] Fixed dialogs' title ellipsis not showing

### PR DESCRIPTION
**Pull request by dcleao**
- For GWT, XUL/GWT and BootstrapJS dialogs

@pentaho/wcag, please, review.

### CODE REVIEW

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 6m 49s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl widgets
```
---
:ok_hand: All tests passed!
 
Tests run: **378**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/gwt-widgets/job/PR-856/1/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Review by soagarwal1 — APPROVED** [review]

File: widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/ruby/globalRuby.css [hunk 1]
```diff
@@ -8948,4 +8950,4 @@ body.bootstrap .modal.confirmationDialog .dialog div.body {
 
 .import-dialog-disclosure-panel .content > tbody > tr > td:nth-child(1) {
   flex: none;
-}
\ No newline at end of file
+}
```
Comment (NJtwentyone) [inline widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/ruby/globalRuby.css:8953]: no newline at end of file.

**Review by NJtwentyone — CHANGES_REQUESTED** [review]
minor change: add back newline at end of file

**Comment by dcleao** [comment]
@NJtwentyone, the change that you mentioned was the new line being added at the end of the file :-)

**Review by singletonc — APPROVED** [review]

File: widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/ruby/globalRuby.css (hunk 1 above)
Comment (NJtwentyone) [reply widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/ruby/globalRuby.css:8953]: mis read the symbol my mistake

**Comment by NJtwentyone** [comment]
> @NJtwentyone, the change that you mentioned was the new line being added at the end of the file :-)

I miss read the symbol, my mistake.

**Review by NJtwentyone — APPROVED** [review]

**Pull request merged**